config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
I can live with that, but what's the downside of using the filter? With the filter, I can easily imagine that it will help certain amount of people from being confused in the future.